### ISSUE TITLE
somthing wrong  in dealing with Dataset

### ISSUE THREAD

**Issue by lp2900** [lwm412/STFGNN-Pytorch#1]
When I run it in another datasets,  it achieve an incredible result. Preds is almost the same with the groud_truth in test_data. And, the groud_truth is changed when self.scaler.inverse_transform is used. Maybe it's not float32? I'm confused

**Comment by lwm412** [comment]
When a new dataset needs to be explored, there are several steps for you to check:
1. Make sure that you set the correct parameters in dataset.json.
2. For the STFGNN model, you should check the stfgnn_dataset.py to make sure the correct adjacency is generated.
3. Make sure you correctly segment the dataset and evaluate the performance of the model (do not confuse the training set with the test set).

**Comment by TudorMarginean** [comment]
I tried the code on a similar dataset, but with just 39 nodes and 1808 time steps - otherwise I followed your implementation. I only changed conv1d to conv2d - maybe this is the problem, because it doesn't learn anything, I'm stuck to a loss of over 200,000 after 70 epochs. But conv1d doesn't work, because it doesn't expect a 4-dimensional shape.
